docs/question/index.md

```diff
@@ -141,104 +176,48 @@ The `question.html` is a template used to render the question to the student. A
 
 The `question.html` is regular HTML, with four special features:
 
-1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values. If you use triple-braces (like `{{{params.html}}}`) then raw HTML is substituted (don't use this unless you know you need it). This is using [Mustache](https://mustache.github.io/mustache.5.html) templating.
+1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values. These values typically are defined by a [question's `server.py`](#custom-generation-and-grading-serverpy).
```

Comment:
I think you should just state that full Mustache substitution is enabled, instead of just this description, which is a little incomplete.

Reply:
I'll add back the Mustache templating link so that this is clear.

Reply:
Added in 70a48696e4fd3c1f41deb53e22204723e1e8e1af.